src/app/modules/gb-side-panel-module/gb-side-panel.component.ts

```diff
@@ -16,8 +18,20 @@ import {SavedCohortsPatientListService} from '../../services/saved-cohorts-patie
   styleUrls: ['./gb-side-panel.component.css']
 })
 export class GbSidePanelComponent {
-
   constructor(public navbarService: NavbarService,
-              public savedCohortsPatientListService: SavedCohortsPatientListService) { }
+              public savedCohortsPatientListService: SavedCohortsPatientListService,
+              public ontologyNavbarService: OntologyNavbarService,
+              public termSearchService: TermSearchService,
+              public renderer: Renderer2) { }
 
+    ngAfterViewInit() {
+      this.termSearchService.searchResultObservable.subscribe(searchResults => {
+        setTimeout(() => {
+          const elems = document.querySelectorAll('.term-search p-header');
```

Comment:
```suggestion
const elems = this.accordionTab.accordion.el.nativeElement.querySelectorAll('p-header'))
```
Here is a suggestion to avoid using `querySelectorAll` on the whole document. Here we select on the current document the accordion tab and then apply `querySelectorAll` on the accordion tab to select the `p-header` children. I tested this locally it seems to work.